### PR TITLE
Add notice to PackageHub enablement in early development

### PR DESCRIPTION
**Pull request by foursixnine**
Save time to reviewers by specifying what's the problem (or a possible cause)

https://openqa.suse.de/t7150168

### CODE REVIEW

**Comment by foursixnine** [comment]
Well, at least test didn't explode :)

File: lib/registration.pm [hunk 1]
```diff
@@ -196,8 +196,13 @@ sub add_suseconnect_product {
     if ($result != 0 && $retry) {
         if ($name =~ /PackageHub/) {
             record_soft_failure 'bsc#1124318 - Fail to get module repo metadata - running the command again as a workaround';
+
         }
-        assert_script_run("SUSEConnect -p $name/$version/$arch $params", $timeout);
+
+        my $fail_message = undef;
+        $fail_message = "PackageHub installation might fail in early development" if ($name =~ /PackageHub/ && check_var('BETA', '1'));
```
Comment (okurz) [inline lib/registration.pm:203]:
```suggestion
        my $fail_message = ($name =~ /PackageHub/ && check_var('BETA', '1')) ? "PackageHub installation might fail in early development" : undef;
```

**Review by okurz — COMMENTED** [review]
+1

File: lib/registration.pm (hunk 1 above)
Comment (foursixnine) [reply lib/registration.pm]: Done

**Review by okurz — APPROVED** [review]

**Review by punkioudi — APPROVED** [review]
In which runs can I see the fail message? Other than that lgtm!

**Comment by okurz** [comment]
> In which runs can I see the fail message? Other than that lgtm!

do you mean a verification run? The message would show up whenever the `assert_script_run` fails. I would not be worried about the format of `assert_script_run`. The open point however would be if the condition is ever met. But for me at this point in code review I care that a regression is unlikely. If the condition is actually met I leave to @foursixnine to prove in production ;)

**Comment by punkioudi** [comment]
Yes, I know when the message will show up, I just asked if there is any verification run

**Comment by foursixnine** [comment]
> Yes, I know when the message will show up, I just asked if there is any verification run

Ideally the VR I provided was supposed to be it, however between me writing the code and pushing the changes, they decided to enable the repository :).

At least the code won't break

**Pull request merged**